### DISCUSSION TITLE
Directus Version 9 licence

### DISCUSSION THREAD

**Discussion by TM61a** [directus/directus#22291] [Feature Requests]
### Summary

I have seen that the licence of directus has changed to BSL.
Is a existing old installation Version 9.22.3 still open source or has old version moved to the new licence also?
from which version does the new license apply?

### Motivation

Old installation have no budget to update

### Detailed Design

.

### Requirements List

.

### Drawbacks

.

### Alternatives

.

### Adoption Strategy

.

**Comment by br41nslug** [comment]
As mentioned in the discord the license was introduced in the 10.0 release. You cannot retro-actively change licenses. Beware that older versions do not receive any updates anymore (security patches or otherwise).

**Comment by TM61a** [comment]
Thanks for clarification.
